packages/atomic/src/index.html

```diff
@@ -40,9 +50,7 @@
         <atomic-tab expression='@author *= "BBC News"'>BBC News</atomic-tab>
       </div>
       <atomic-context-provider context='{"test":"value","test2":"value2"}'></atomic-context-provider>
-      <atomic-search-box>
-        <span slot="submit-button">Go!</span>
-      </atomic-search-box>
+      <atomic-search-box></atomic-search-box>
```

Comment:
I know we said that we are not going to use `slots` for the first release.

However, if, down the line, we want to start adding them in some specific places, would there be a way to make the localized string available for the end users in their slots ?

Or are we painting ourselves in some corner/introducing some impossible to bypass limitation ?

Reply:
Eventually we would create  an atomic-component that displays localized strings e.g. `atomic-text`

Reply:
would simply interface the i18next.t() method, which can take multiple arguments